controller/environment.go

```diff
@@ -80,7 +82,9 @@ func (c *EnvironmentController) Create(ctx *app.CreateEnvironmentContext) error
 
 		env, err = appl.Environments().Create(ctx, &newEnv)
 		if err != nil {
-			return errs.Wrapf(err, "failed to create space: %s", newEnv.Name)
+			log.Debug(ctx, map[string]interface{}{"err": err},
```

Comment:
Should be `log.Error()`

Reply:
done commit [a1a7b87](https://github.com/fabric8-services/fabric8-env/pull/2/commits/a1a7b877ef7fbfc5f8cebceac836a4fd086700dc)